tests/complete_run.py

```diff
@@ -14,7 +14,7 @@ def test_complete_run(self):
         actual_images_dir = run_lcrc(".")
 
         # The expected_images_file lists all images we expect to compare.
-        expected_images_file = "/lcrc/group/e3sm/public_html/e3sm_diags_test_data/unit_test_complete_run/expected/image_list_all_sets.txt"
+        expected_images_file = "/lcrc/group/e3sm/public_html/e3sm_diags_test_data/unit_test_complete_run/expected/image_list_all_sets2.txt"
```

Comment:
I no longer have write permissions to `/lcrc/group/e3sm/public_html/e3sm_diags_test_data/unit_test_complete_run/expected/image_list_all_sets.txt` -- I'd prefer to edit this file rather than pointing to a new file.